lib/src/color_picker.dart

```diff
@@ -272,26 +272,31 @@ class _ColorPickerState extends State<ColorPicker> {
     );
   }
 
-  Widget _buildDropdown2() {
-    return SizedBox(
-      height: 38,
-      child: DecoratedBox(
-        decoration: BoxDecoration(
-          border: Border.all(color: Theme.of(context).dividerColor),
-          borderRadius: const BorderRadius.all(
-            Radius.circular(3.0),
+  Widget _buildDropdownPortraitMode() {
+    return Padding(
```

Comment:
Should we use a `Container` instead?

Reply:
I think that `Container` might be more expensive than Padding and SizedBox (when I look at their code).
Our code would look better though.
Change it?

Reply:
We have a `DecoratedBox` here too. Actually `Container` will only affects the box constraints, it won't be too many differences when it involved with performance.